src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -33,46 +37,38 @@ async def handle_message(self, msg):
                 )
             )
 
-    async def set_position_control_mode(self, control_mode):
+    async def set_position_control_mode(self, control_mode: int) -> None:
         """Set the control loop mode.
 
         When in closed-loop mode, position is maintained by a feedback signal
         from the piezo actuator. This is only possible when using actuators
         equipped with position sensing.
 
         :param control_mode: 0x01 for Open Loop (no feedback).
-
             0x02 for Closed Loop (feedback employed).
-
             0x03 for Open Loop Smooth.
-
             0x04 for Closed Loop Smooth.
         """
         await self.send(
-            Message(MGMSG.PZ_SET_POSCONTROLMODE, param1=1, param2=control_mode)
+            Message(MGMSG.PZ_SET_POSCONTROLMODE, param1=_CHANNEL, param2=control_mode)
```

Comment:
```suggestion
            Message(MGMSG.PZ_SET_POSCONTROLMODE, param1=Tpz._CHANNEL, param2=control_mode)
```
Use class constants.